package-managers/nuget.md

```diff
@@ -54,36 +54,69 @@ inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual
 
 ### JetBrains Rider
 
-#### Public feed
+The first step in connecting Rider to a MyGet feed is adding it as a package source. We can do this using NuGet.exe (via good old [NuGet.config](https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#package-source-sections), or from within Rider. From the **NuGet tool window** (<kbd>Alt</kbd> + <kbd>7</kbd> will bring this right up), open the **Sources** tab. This will show us all of the NuGet configuration files that are in play, and a list of all feeds configured.

```

Comment:
Perhaps use back-ticks around `nuget.exe`? (didn't we settle on lowercase `nuget` when referring to the CLI?)

Reply:
`NuGet.exe` I believe